modules/tools/graph_generator/graph_creator.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+#
+ # Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ #
+ #   WSO2 Inc. licenses this file to you under the Apache License,
+ #   Version 2.0 (the "License"); you may not use this file except
+ #   in compliance with the License.
+ #   You may obtain a copy of the License at
+ #
+ #      http://www.apache.org/licenses/LICENSE-2.0
+ #
+ #   Unless required by applicable law or agreed to in writing,
+ #   software distributed under the License is distributed on an
+ #   "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ #   KIND, either express or implied.  See the License for the
+ #   specific language governing permissions and limitations
+ #   under the License.
+#
+FILES=PATH-TO-DATAFILES DIRECTORY/*          #Path for the directory where text files are stored/*
+DATAFILES=PATH-TO-PROJECT-FOLDER/graph_generator/DataFiles/   #Path for the directory where data files should be stored/
+GRAPHS=PATH-TO-PROJECT-FOLDER/graph_generator/Graphs/         #Path for the directory where .png files should be stored/
+
+new_file=PATH-TO-PROJECT-FOLDER/graph_generator/searched.txt  #New file path for the intermediate operations
+new_file1=PATH-TO-PROJECT-FOLDER/graph_generator/searched1.txts
```

Comment:
Shall we give dome meaningful names to these files. What is the difference between search1 and search2?